spec/units/user_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'user'
+
+describe User do
+  describe '.create' do
+    it 'creates a new user' do
+      user = create_new_user
```

Comment:
Fine to use a test helper 'create_new_user' when you are just creating a user to support testing of some other code - but if the thing that I am testing is the code in the test helper, I would want the code for that to appear in the test itself, otherwise I have to look in another file to see what I am testing.

Also, it feels like this is testing state not behaviour. Have you come across that idea yet? Basically, there is no point in testing the initial state of an object because you are implicitly testing this by testing behaviour elsewhere. For example, the test 'returns a user given a correct username and password' will fail if either the id or the password is not initialised correctly, which makes this test redundant.